Update tab-sso-graph-api.md (#7337)
MicrosoftDocs/msteams-docs: msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md

1. Use token cache serialization in MSAL.NET to cache the new access token for multiple, if required.

> [!IMPORTANT]

> > * As a best practice for security, always use the server-side code to make Microsoft Graph calls or other calls that require passing an access token. You must not return the OBO token to the client because it enables the client to make direct calls to Microsoft Graph. This helps protect the token from being intercepted or leaked.

Reviewer: If you want to link back to the Identity talks discussing this topic, you can find that document here:
https://learn.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow#middle-tier-access-token-request

> > - As a best practice for security, always use the server-side code to make Microsoft Graph calls, or other calls that require passing an access token. Never return the OBO token to the client to enable the client to make direct calls to Microsoft Graph. This helps protect the token from being intercepted or leaked.
> > 
> > - Do not use `notifySuccess` result to return token information to the parent page. Use localStorage to avoid passing the token via notifySuccess; instead we send the item key. 

Reviewer 2: ```suggestion
> * As a best practice for security, always use the server-side code to make Microsoft Graph calls or other calls that require passing an access token. You must not return the OBO token to the client because it enables the client to make direct calls to Microsoft Graph. This helps protect the token from being intercepted or leaked.
> 
> * Don’t use `notifySuccess` result to return the token information to the parent page. Use `localStorage` to avoid passing the token via `notifySuccess`, instead the item key is sent. 
```
